SPECS/bzip2/bzip2.spec

```diff
@@ -39,8 +37,6 @@ This package contains minimal set of shared bzip2 libraries.
 %setup -q
```

Comment:
Should we probably use `%auto_setup` here  instead.

Reply:
fixed